using-the-website/staking.md

```diff
@@ -38,6 +38,6 @@ Staking allows you to earn OHM passively via auto-compounding. By staking your O
 
 **ROI (5-day rate)** estimates how much your sOHM balance will increase after 5 days, if the upcoming rebase rate stays the same during this period. For example, if you stake 100 OHM and the rate is 10.0250%, your sOHM balance would increase from 100 to 110.025 after 5 days.
 
-**Upcoming APY** tells you the annualized rate of the current epoch. Since sOHM rebases exponentially, we display a compounding rate. You can determine this number for yourself with the calculation \( \(1 + rebase\) ^ \(365 \* 3\) \) - 1
+**Current APY** tells you the annualized rate of the current epoch. Since sOHM rebases exponentially, we display a compounding rate. You can determine this number for yourself with the calculation \( \(1 + rebase\) ^ \(365 \* 3\) \)
```

Comment:
I am not sure the formula is correct to calculate the APY. Somebody else verify please.

Reply:
I think the -1 part of the equation is to subtract the initial principal so that you get the % _increase_ from your initial principal. To calculate your balance at the end of the year, you simply add this % to your initial principal. If we remove the -1 part, we would multiply our initial principal with the % value to get the final balance. In other words: P(APY -1) + P = P(APY)

Reply:
I compare the rebase rate and the APY at the current epoch, which is 0.6399% and 108041.00% respectively, and found that the APY calculation does not take into account the -1 part. So I think we could remove it from the definition.

To emphasize again, there is nothing wrong with the -1 part, it only affects how we calculate the balance at the end of the year. And as it stands, the APY displayed on OlympusDAO website does not take into account the -1 part.

Reply:
I am ok with removing the formula here if it would just lead to confusion otherwise. If we want to have the APY calculation we should put it in the equations section and have it explained separately.

Reply:
```suggestion
**Current APY** tells you the annualized rate of return based on the upcoming rebase rate. It takes into account the effect of compounding since sOHM rebases exponentially.
```